jekyll/_cci2/container-runner-installation.adoc

```diff
@@ -77,6 +77,49 @@ agent:
+
+The ability to rerun a job with SSH is in open preview for CircleCI container runner. To enable this optional feature, follow the instructions below after reading through the considerations. Rerunning jobs with SSH allows you to troubleshoot problems through inspecting log files, running processes, and directory paths.
+
+[CAUTION]
+====
+**Retry with SSH considerations**
+
+- Task-agent runs an embedded SSH server and agent on a dedicated port when the “Rerun job with SSH” option is activated. This feature will not affect any other SSH servers or agents on the cluster that container runner is installed on.
+
+- The SSH server is configured for public key authentication. Anyone with permission to initiate a job can rerun it with SSH. However, only the user who initiated the rerun will have their SSH public keys added to the server for the duration of the SSH session.
+
+- Rerunning a job with SSH will hold the job open for **two hours** if a connection is made to the SSH server, or **10 minutes** if no connection is made, unless canceled. While in this state, the job is counted against an organization’s concurrency limit, and the task-agent will be unavailable to handle other jobs. Therefore, it is recommended to cancel an SSH rerun job explicitly (through the web UI or CLI) when you have finished debugging.
+
+- When establishing a SSH connection, you may experience a warning relating to a host change, due to the reuse of a previously used port to connect to a different pod. RSA host fingerprints can be cross-referenced with the output in the console for verification.
+====
+
+The Rerun job with SSH feature makes use of the link:https://gateway-api.sigs.k8s.io/[Gateway API] to facilitate SSH access from outside your cluster. To enable this feature, Gateway API resources must first be provisioned on your cluster, and the CircleCI container agent configuration must be updated.
+
+
+NOTE: **Supported Gateway API implementations:**
+CircleCI has tested, and currently supports link:https://gateway.envoyproxy.io/latest/user/tcp-routing/[Envoy Gateway] as an implementation for the link:https://gateway-api.sigs.k8s.io/[Gateway API]. Other Gateway API implementations that support TCPRoute resources are likely compatible, but these implementations have not been tested, and may not be compatible. Our recommendation is to use the link:https://gateway.envoyproxy.io/latest/user/tcp-routing/[Envoy Gateway chart] as your implementation.
+
+
+==== 1. Install Envoy Gateway to your cluster
+First, install the Envoy Gateway chart and wait for Envoy Gateway to become available, as defined in the
+link:https://gateway.envoyproxy.io/latest/user/tcp-routing/[Envoy Gateway TCP Routing installation document].
+
+
+==== 2. Enable the Rerun job with SSH feature
+Once all Gateway API prerequisites are installed and available, add the `agent.ssh.enabled = “true”` parameter to the container runner configuration, and redeploy the updated manifest.
```

Comment:
@christian-stephen for the "redeploy the updated manifest" part can we give them a command in a snippet for that to make it easy?

Reply:
Yeah, for sure. Following the Helm install command [here](https://circleci.com/docs/container-runner-installation/#container-runner-installation), they can do a Helm upgrade for the re-deploy:

```
helm upgrade container-agent container-agent/container-agent -n circleci -f values.yaml
```